ruleguard.rules.go

```diff
@@ -322,6 +322,15 @@ func joinpath(m dsl.Matcher) {
 		Report(`did you mean path.Join() or filepath.Join() ?`)
 }
 
+func rewriteSplitN(m dsl.Matcher) {
+	m.Match(`$x := strings.SplitN($y, $delim, 2)[0]`).
```

Comment:
You can add this as it's commonly used

```suggestion
	m.Match(`$x := strings.SplitN($y, $delim, 2)[0]`, `$x := strings.SplitN($y, $delim, -1)[0]`).
```

Reply:
Is there something that catches `Split($y, $delim)[0]`

It's commonly used too

Reply:
Sure, in that case I will also add `$x := strings.Split($y, $delim)[0]` too then

Reply:
Done